saleor/graphql/campaign/enums.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+
+
+class IndustryEnum(Enum):
+    ACADEMIC = "ACADEMIC"
+    ACCOUNTING = "ACCOUNTING"
+    ADVERTISING = "ADVERTISING"
+    AEROSPACE = "AEROSPACE"
+    AGRICULTURE = "AGRICULTURE"
+    ARTS_CULTURE = "ARTS_CULTURE"
+    AUTOMOTIVE = "AUTOMOTIVE"
+    AVIATION = "AVIATION"
+    BANKING = "BANKING"
+    BUILDING_CONSTRUCTION = "BUILDING_CONSTRUCTION"
+    CHARITY_AND_NOT_FOR_PROFIT = "CHARITY_AND_NOT_FOR_PROFIT"
+    COMMUNICATIONS = "COMMUNICATIONS"
+    EDUCATION = "EDUCATION"
+    ENERGY = "ENERGY"
+    ENGINEERING = "ENGINEERING"
+    ENTERTAINMENT = "ENTERTAINMENT"
+    FACILITY_MANAGEMENT = "FACILITY_MANAGEMENT"
+    FASHION = "FASHION"
+    FAST_MOVING_CONSUMER_GOODS = "FAST_MOVING_CONSUMER_GOODS"
+    FINANCE = "FINANCE"
+    FOOD = "FOOD"
+    GENERIC = "GENERIC"
+    GOVERNMENT_AND_PUBLIC_SECTOR = "GOVERNMENT_AND_PUBLIC_SECTOR"
+    HEALTH_AND_SAFETY = "HEALTH_AND_SAFETY"
+    HEALTHCARE = "HEALTHCARE"
+    HOSPITALITY = "HOSPITALITY"
+    INFORMATION_TECHNOLOGY = "INFORMATION_TECHNOLOGY"
+    INSURANCE = "INSURANCE"
+    LANGUAGE = "LANGUAGE"
+    LEGAL = "LEGAL"
+    LEISURE_AND_SPORT = "LEISURE_AND_SPORT"
+    LOGISTICS_SUPPLY_CHAIN = "LOGISTICS_SUPPLY"
+    MANUFACTURING = "MANUFACTURING"
+    MARITIME = "MARITIME"
+    MEDIA = "MEDIA"
+    MULTILINGUAL = "MULTILINGUAL"
+    PHARMACEUTICALS = "PHARMACEUTICALS"
+    PROPERTY_AND_HOUSING = "PROPERTY_AND_HOUSING"
+    REAL_ESTATE = "REAL_ESTATE"
+    RECRUITMENT = "RECRUITMENT"
+    RETAIL = "RETAIL"
+    SCIENCE_RESEARCH = "SCIENCE_RESEARCH"
+    SUSTAINABILITY = "SUSTAINABILITY"
+    TELECOMMUNICATIONS = "TELECOMMUNICATIONS"
+    TRANSPORT_AND_RAIL = "TRANSPORT_AND_RAIL"
+    TRAVEL_AND_TOURISM = "TRAVEL_AND_TOURISM"
+
+
+class SeniorityEnum(Enum):
+    TRAINEE = "TRAINEE"
```

Comment:
seniorities need to be from below. labels are from the `name.value` where languageCode is `en_EN`. raw value passed to the order needs to be from the `id` key

```
[
    {
        "id": 1,
        "name": [
            {
                "languageCode": "de_DE",
                "value": "Geschäftsführer / C-Level"
            },
            {
                "languageCode": "en_GB",
                "value": "Executive/Director"
            },
            {
                "languageCode": "nl_NL",
                "value": "Executive/Director"
            }
        ]
    },
    {
        "id": 2,
        "name": [
            {
                "languageCode": "de_DE",
                "value": "Manager"
            },
            {
                "languageCode": "en_GB",
                "value": "Manager"
            },
            {
                "languageCode": "nl_NL",
                "value": "Manager"
            }
        ]
    },
    {
        "id": 3,
        "name": [
            {
                "languageCode": "de_DE",
                "value": "Professional"
            },
            {
                "languageCode": "en_GB",
                "value": "Mid-Senior level"
            },
            {
                "languageCode": "nl_NL",
                "value": "Professional"
            }
        ]
    },
    {
        "id": 4,
        "name": [
            {
                "languageCode": "de_DE",
                "value": "Berufseinsteiger"
            },
            {
                "languageCode": "en_GB",
                "value": "Entry level/Graduate"
            },
            {
                "languageCode": "nl_NL",
                "value": "Entry level/Graduate"
            }
        ]
    },
    {
        "id": 5,
        "name": [
            {
                "languageCode": "de_DE",
                "value": "Student/Trainee"
            },
            {
                "languageCode": "en_GB",
                "value": "Student/Trainee"
            },
            {
                "languageCode": "nl_NL",
                "value": "Student/Trainee"
            }
        ]
    }
]
```